<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OnlineMusicWidget</class>
 <widget class="QWidget" name="OnlineMusicWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OnlineMusicWidget</string>
  </property>
  <widget class="QPushButton" name="Mini">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>0</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;
color:white;
}
QPushButton:hover
{
background-repeat:no-repeat;
background-position:center center;
}
QPushButton:pressed
{
background-repeat:no-repeat;
background-position:center center;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix2/image/minimizing.png</normaloff>:/new/prefix2/image/minimizing.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Title">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>201</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px;
coloe:rgb(225,225,225);
background-color:rgba(225,225,225,150);</string>
   </property>
   <property name="text">
    <string>搜索歌曲:</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix2/image/music.png</normaloff>:/new/prefix2/image/music.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Search">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>60</y>
     <width>201</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px;
coloe:rgb(225,225,225);
background-color:rgba(225,225,225,150);</string>
   </property>
   <property name="text">
    <string>SEARCH</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix2/image/search.png</normaloff>:/new/prefix2/image/search.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="InputSong">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>621</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>1061</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>291</width>
     <height>541</height>
    </rect>
   </property>
   <property name="title">
    <string>在线音乐</string>
   </property>
   <widget class="QTextEdit" name="OnlineSongList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>271</width>
      <height>511</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SimSun-ExtB</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:14px;
coloe:rgb(225,225,225);
background-color:rgba(225,225,225,150);</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>140</y>
     <width>471</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14px;
coloe:rgb(225,225,225);
background-color:rgba(225,225,225,150);</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>491</width>
     <height>541</height>
    </rect>
   </property>
   <property name="title">
    <string>歌词</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_3">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>140</y>
     <width>251</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14px;
coloe:rgb(225,225,225);
background-color:rgba(225,225,225,150);</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>120</y>
     <width>271</width>
     <height>541</height>
    </rect>
   </property>
   <property name="title">
    <string>播放历史</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Skin">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;
color:white;
}
QPushButton:hover
{
background-repeat:no-repeat;
background-position:center center;
}
QPushButton:pressed
{
background-repeat:no-repeat;
background-position:center center;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix2/image/skin.png</normaloff>:/new/prefix2/image/skin.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Documents">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>730</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;
color:white;
}
QPushButton:hover
{
background-repeat:no-repeat;
background-position:center center;
}
QPushButton:pressed
{
background-repeat:no-repeat;
background-position:center center;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix2/image/Document.png</normaloff>:/new/prefix2/image/Document.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Play">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>730</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;
color:white;
}
QPushButton:hover
{
background-repeat:no-repeat;
background-position:center center;
}
QPushButton:pressed
{
background-repeat:no-repeat;
background-position:center center;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix2/image/24gl-play.png</normaloff>:/new/prefix2/image/24gl-play.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Pause">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>730</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;
color:white;
}
QPushButton:hover
{
background-repeat:no-repeat;
background-position:center center;
}
QPushButton:pressed
{
background-repeat:no-repeat;
background-position:center center;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix2/image/stop.png</normaloff>:/new/prefix2/image/stop.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Stop">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>730</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;
color:white;
}
QPushButton:hover
{
background-repeat:no-repeat;
background-position:center center;
}
QPushButton:pressed
{
background-repeat:no-repeat;
background-position:center center;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix2/image/stop2.png</normaloff>:/new/prefix2/image/stop2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Piror">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>730</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;
color:white;
}
QPushButton:hover
{
background-repeat:no-repeat;
background-position:center center;
}
QPushButton:pressed
{
background-repeat:no-repeat;
background-position:center center;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix2/image/piror.png</normaloff>:/new/prefix2/image/piror.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Volumn">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>730</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;
color:white;
}
QPushButton:hover
{
background-repeat:no-repeat;
background-position:center center;
}
QPushButton:pressed
{
background-repeat:no-repeat;
background-position:center center;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix2/image/volumn.png</normaloff>:/new/prefix2/image/volumn.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>680</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>680</y>
     <width>881</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_2">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>740</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;
color:white;
}
QPushButton:hover
{
background-repeat:no-repeat;
background-position:center center;
}
QPushButton:pressed
{
background-repeat:no-repeat;
background-position:center center;
}</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>36</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>670</y>
     <width>1071</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="next">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>60</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;
color:white;
}
QPushButton:hover
{
background-repeat:no-repeat;
background-position:center center;
}
QPushButton:pressed
{
background-repeat:no-repeat;
background-position:center center;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/new/prefix2/image/next.png</normaloff>:/new/prefix2/image/next.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Close">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>0</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;
color:white;
}
QPushButton:hover
{
background-repeat:no-repeat;
background-position:center center;
}
QPushButton:pressed
{
background-repeat:no-repeat;
background-position:center center;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix2/image/close2.png</normaloff>:/new/prefix2/image/close2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <zorder>groupBox_3</zorder>
  <zorder>Skin</zorder>
  <zorder>groupBox_5</zorder>
  <zorder>groupBox_4</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>Mini</zorder>
  <zorder>Title</zorder>
  <zorder>Search</zorder>
  <zorder>InputSong</zorder>
  <zorder>textEdit_2</zorder>
  <zorder>textEdit_3</zorder>
  <zorder>Documents</zorder>
  <zorder>Play</zorder>
  <zorder>Pause</zorder>
  <zorder>Stop</zorder>
  <zorder>Piror</zorder>
  <zorder>Volumn</zorder>
  <zorder>lcdNumber</zorder>
  <zorder>horizontalSlider</zorder>
  <zorder>horizontalSlider_2</zorder>
  <zorder>Close</zorder>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
